src/map/status.cpp

```diff
@@ -9223,7 +9223,7 @@ std::vector<e_race2> status_get_race2(struct block_list *bl)
  */
 int status_isdead(struct block_list *bl)
```

Comment:
Can we use ref and return a bool instead?

Reply:
it's that?

Reply:
if change status_getdata  to ref is bad idea, say and i revert it  @Atemo